contrib/fparser/fparser_ad.cc

```diff
@@ -601,7 +618,11 @@ bool FunctionParserADBase<Value_t>::JITCompileHelper(const std::string & Value_t
   // generate a sha1 hash of the current program and the Value type name
   SHA1 *sha1 = new SHA1();
   char result[41];
+  // 1. version string
+  sha1->addBytes("2.0", 3);
```

Comment:
I'm adding a version string here to force a rebuild of previously cached JIT objects.